lib/zss.rb

```diff
@@ -5,6 +5,8 @@
 
 require 'logger_facade'
 
+require_relative 'zss/require'
```

Comment:
For the some reason why we don't require service and client library we should not require this by default. The client of the library should require the needed modules. What do you think?